src/ui/base-page-storyEdit.js

```diff
@@ -16,6 +16,40 @@ https://raw.githubusercontent.com/fluid-project/sjrk-story-telling/master/LICENS
         pageSetup: {
             hiddenEditorClass: "hidden"
         },
+        model: {
+            // the initial state is only the editor ui showing.
+            // in much the same way as within the editor, the visibility
+            // of the editor and previewer are mutually exclusive and
+            // represented by a single model value, for the time being
+            editorVisible: true
```

Comment:
As below, with the use of "oldValue" which @cindyli has noted below. Rather than applying this historyful gimmick, you should have a properly organised model with separate flags for "editorVisible" and "previewerVisible". Connect them with a relay - presumably this is the situation which gave rise to the legendary standup update "Infusion doesn't have an inversion relay" : P
Looking at the impl for the fluid.transforms.linearScale transform which could have been hacked into service, I see that @kaspermarkus unhelpfully put some type checks into it, so I suggest you just apply
````
    sjrk.storyTelling.not = function (value) {
          return !value;
     };
````
and specify this as the relay rule.

Reply:
For extra helpfulness, please contribute this to Infusion as fluid.transforms.not together with its self-inverse definition and a couple of small test cases :)

Reply:
Indeed, this is precisely the situation that gave rise to my comments ;) I began to work on such a transform, but we decided that for the sake of expedience I would shelve that work and Just Get It Working™ with this binary view-control method, and come back to this when there is less pressing work to be done.

I've achieved a less confusing solution via [the method I proposed a couple review items above](https://github.com/fluid-project/sjrk-story-telling/pull/56#discussion_r377315295). It's a bit of a middle ground between using `{change}.oldValue` and re-implementing the model relay I once had (with either a free transform calling this new `not` function or as a full-fledged inversion transform). Not ideal, but perhaps good enough for now? I can file a Jira to make sure we repay this technical debt.